recipes/amazon.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: osquery
+# Recipe:: amazon
+#
+# Copyright 2019, Dwolla Inc.
```

Comment:
Can you remove this copyright? It goes against the Apache 2 License

Reply:
Yep! No problem.